tests/bluetooth/tester/src/btp.c

```diff
@@ -90,25 +87,27 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 	while (1) {
 		const struct btp_handler *btp;
 		struct btp_buf *cmd;
+		struct btp_hdr *hdr;
 		uint8_t status;
 		uint16_t rsp_len = 0;
 		uint16_t len;
 
 		cmd = k_fifo_get(&cmds_queue, K_FOREVER);
+		hdr = (struct btp_hdr *)cmd->data;
```

Comment:
Should we have an 
```suggestion
		hdr = (struct btp_hdr *)cmd->data;
		__ASSERT_NO_MSG(hdr->len <= BTP_DATA_MAX_SIZE);
```

Reply:
actually, I'm not sure if this should be assert as this means we assert on invalid command,  I'll make this runtime check and simply return error (like we do for unknown command